cmd/gaiacli/swagger-ui/swagger.yaml

```diff
@@ -0,0 +1,2461 @@
+---
+swagger: "2.0"
+info:
+  version: "3.0"
+  title: Gaia-Lite for Cosmos
+  description: A REST interface for state queries, transaction generation and broadcasting.
+tags:
+  - name: ICS0
```

Comment:
related to updates from https://github.com/cosmos/cosmos-sdk/pull/4429, will we always have to update the swagger file in both repos ?

Reply:
thanks for pointing this out, can be removed from here :)